application/feature/characters/list/impl/src/main/java/com/stepango/blockme/feature/characters/list/impl/data/datasource/CharactersPageDataSource.kt

```diff
@@ -41,14 +41,14 @@ const val PAGE_MAX_ELEMENTS = 50
 open class CharacterPageDataSource @Inject constructor(
     private val repository: MarvelRepository,
```

Comment:
I think MarvelRepository should use `ICharacter` type too.
And `CharacterMapper`  we can move into core part.